docs/omero-release-process.rst

```diff
@@ -0,0 +1,111 @@
+ - Add an entry to `history.rst <https://github.com/ome/openmicroscopy/blob/develop/history.rst>`_.
+
+After committing the changes, a signed tag must be created for the released version
+using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" vx.y.z
+
+Both the `develop` branch as well as the tag must be pushed upstream::
+
+    $ git push origin develop vx.y.z
+
+This will trigger GitHub action builds, generate artifacts, create a GitHub release and upload
+the artifacts to the release on GitHub.
+An hourly cron job runs on our virtual machine and copy the artifacts published on Github under `OMERO Downloads`_.
+
+Docker container release
+------------------------
+
+Before making a new release, merge all contributions on the `master` branch (rare).
+
+When the artifacts are available under `OMERO Downloads`_, you can release the `OMERO Server Docker`_.
+Bump the version in `Dockerfile <https://github.com/ome/omero-server-docker/blob/master/Dockerfile>`_ to the latest release version.
+
+After committing the changes, a signed tag must be created for the released version
+using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" x.y.z
+
+Before pushing, make sure that the artifacts have been copied under `OMERO Downloads`_.
+Both the `master` branch as well as the tag must be pushed upstream::
+
+    $ git push origin master x.y.z
+
+
+Documentation release
+---------------------
+
+Before making a new release, merge all contributions on the `develop` branch.
+
+A `GitHub action <https://github.com/ome/omero-documentation/blob/develop/.github/workflows/update.yml>`_ is run daily to synchronize the documentation with changes made in various repositories
+e.g. ``omero-install``, ``omeroweb-install``, ``openmicroscopy``, and bump the version of various components e.g. ``omero-web``. The action will automatically open a Pull Request if some changes happened. The Pull Request must be merged before the release.
+If you wish to publish the documentation the same day as the source code release, the easiest option is to 
+modify the format of the cron job in the `update.yml GitHub action <https://github.com/ome/omero-documentation/blob/develop/.github/workflows/update.yml>`_ and merge the Pull Request opened by the action.
+
+After fetching the `develop` branch, revert the change to the cron job made above.
+A signed tag must be created for the released version using :command:`git tag -s`::
+
+    $ git tag -s -m "Tag version x.y.z" vx.y.z
+
+Both the `develop` branch as well as the tag must be pushed upstream::
+
+    $ git push origin develop vx.y.z
+
+The documentation is built and published on `Read The Docs`_.
```

Comment:
Maybe link to https://readthedocs.org/projects/omero/builds/ (instead of the Read_The_Docs link)?

Reply:
It is already the case